### ISSUE TITLE
Minimal version of swig?

### ISSUE THREAD

**Issue by mlindauer** [automl/random_forest_run#18]
Hi,
on the nemo cluster, swig is installed in version 2.0.10 and the installation fails:

```
    swig -python -c++ -I${CMAKE_SOURCE_DIR}/include -I./include -o pyrfr/regression_wrap.cpp pyrfr/regression.i
    ./include/rfr/trees/binary_fanova_tree.hpp:329: Error: Syntax error in input(3).
```

On our machines, swig is running in version 3.0. 
So, I wanted ask whether there is a minimal required version of SWIG for pyrfr?

Cheers,
Marius

**Comment by mlindauer** [comment]
btw: in case someone has a problem with swig and uses anaconda, here is an easy solution: 
`conda install swig`

**Comment by sfalkner** [comment]
Yes you will need Swig >= 3.0. They added a lot of C++11 support in that version.
Thanks for the pointer to the swig-anaconda problem.

**Comment by sfalkner** [comment]
fixed via 170cfc86abc74a9bac834731270446b43571efcf
